NEWS.md

```diff
@@ -6,6 +6,8 @@
 
 ## Improvements and bug fixes
 
+* `card_header()` is now flex by default and includes a `gap` parameter to better support complex header layouts. (#1253)
```

Comment:
```suggestion
* `card_header()` is now flex by default and includes a `gap` argument to better support complex header layouts. (#1253)
```

_parameter_ is technically the more correct word, but in R we generally refer to these as arguments, at least here in our NEWS file (see a few lines down). In part because the section header in the R help docs is named "Arguments".